roles/gaia/defaults/main.yml

```diff
@@ -3,11 +3,12 @@
 gaiad_user: gaia
 gaiad_user_home: "/home/{{gaiad_user}}"
 gaiad_home: "{{gaiad_user_home}}/.gaiad"
-gaiad_home_autoclear: false
-gaiad_version: v5.0.5
+gaiad_home_autoclear: true
```

Comment:
Setting that here will make it clear the gaiad directory all the time when run. I think it would be better to use `--extra-vars "gaiad_home_autoclear=true"` to define it when needed.

Reply:
I agree that autoclear should probably be opt-in. We could add the `--extra-vars` thing to the README to make it easier to copy-paste. :P